MODMARK.md

```diff
@@ -86,4 +79,49 @@ Example 5: module with arguments
+|----------------------------|--------------------------------|--------------------|
+| ```**bold**```             | <strong>bold</strong>          | yes                |
+| ```//italic//```           | <em>italic</em>                | yes                |
+| ```__subscript__```        | <sub>subscript</sub>           | yes                |
+| ```^^superscript^^```      | <sup>superscript</sup>         | yes                |
+| <code>\``verbatim``</code> | verbatim                       | no                 |
+| ```==underlined==```       | <ins>underlined</ins>          | yes                |
+| ```~~strikethrough~~```    | <strike>strikethrough</strike> | yes                |
+| ```$$math x^2$$```         | <pre>math x<sup>2<sup></pre>   | no                 |
+
+Tags will always match the first possible closing tag: `**bold***` will have the last star outside the bold tag. As described above, a backslash escapes *one character*, so `**bold\***` will make the first star in the second cluster be just the `*` character, and thus be put inside the bold tag.
+
+## Headings
+
+Headings may occur as the first line of a paragraph, or by themselves, and start with a number of hashtags `#`. One hashtag corresponds to a heading at level 1, two corresponds to a heading at level 2 etc. Any space before the content of the heading will be stripped. Note that while ModMark supports any heading level, not all output formats does. For example, HTML only has headings up to level 6, and LaTeX only supports headings up to level 3 (subsubsection).
+
+| Source                    | Output                   |
+|---------------------------|--------------------------|
+| ```# heading level 1```   | <h1>heading level 1</h1> |
+| ```## heading level 2```  | <h2>heading level 2</h2> |
+| ```### heading level 3``` | <h3>heading level 3</h3> |
+
+## Smart punctuation
+
+Smart punctuation is character sequences that gets replaced by a different character, or encasement in quotes where the quotes gets switched out for left and right quotes. `...` will get replaced by the ellipsis character &hellip;, and "abc" will be replaced with &ldquo;abc&ldquo;. Escaping also applies to these, `\...` will not become an ellipsis and `\....` will become a dot followed by an ellipsis.
+
+| Source                | Output                      |
+|-----------------------|-----------------------------|
+| ```"double quotes"``` | &ldquo;double quotes&ldquo; |
+| ```'single quotes'``` | &lsquo;single quotes&rsquo; |
+| ```a -- b```          | a &ndash; b                 |
+| ```a --- b```         | a &mdash; b                 |
+| ```ellipsis...```     | ellipsis&hellip;            |
+
+*Note: character sequences that could technically be broken up into smaller sequences for smart punctuation will instead be used as is. This means that `----` is not parsed as two endashes, but rather one emdash followed by a normal dash.*
```

Comment:
I'm not sure this is how we actually implemented it. I think the result here is simply four dashes.

Reply:
Fixed!

Reply:
I changed this to a text explaining what is actually happening. You are correct in that four dashes becomes four dashes